baybe/searchspace/core.py

```diff
@@ -300,7 +303,7 @@ def transform(
 
 def validate_searchspace_from_config(specs: dict, _) -> None:
```

Comment:
Just a general thing that I just observed: Is there a specific reason why this function is not part of `baybe.searchspace.validation`?

Reply:
not sure, the other config valdiator `validate_searchspace_from_config` is also not in there but in `.core`

these two functions dont perform attrs-like validation like the content of `.validation` but perform cattrs structuring with reduced content (with the aim to be used for validation), so not sure whether that mens they should go to `.validation`

Reply:
But this means that there is a difference between stuff in validate and the stuff here. Then this is fine for me.